docs: mention `isLoopFinished()` stop condition in context of agent loops (#14539)
vercel/ai: content/docs/07-reference/01-ai-sdk-core/16-tool-loop-agent.mdx

      type: 'StopCondition | StopCondition[]',
      isOptional: true,
      description:

>         'Condition(s) for ending the agent loop. Default: isStepCount(20). Use `isLoopFinished()` to let the agent run until all tool calls have completed, but beware of potential runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',

Reviewer: how will the url render in the docs?

>         'Condition(s) for ending the agent loop. Default: isStepCount(20). Use `isLoopFinished()` to let the agent run until all tool calls have completed, but pair it with a maximum step limit to avoid runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',

Author: ```suggestion
        'Condition(s) for ending the agent loop. Default: isStepCount(20). Use `isLoopFinished()` to let the agent run until all tool calls have completed, but beware of potential runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',
```
